ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -105,6 +105,7 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
+      HYPERSHIFT_HC_RELEASE_IMAGE: quay.io/openshift-release-dev/ocp-release:4.11.0-rc.7-x86_64
```

Comment:
could you explain a little bit why we use a rc build, rather than a latest release build ?

Reply:
yes, I think we could run this CI for the latest 4.11. It is better for the e2e.

Reply:
okay, i close this PR, thanks